ts/Grid/Core/Table/Header/TableHeader.ts

```diff
@@ -113,11 +113,22 @@ class TableHeader {
             return;
         }
 
+        // Render regular, multiple level rows.
         for (let i = 0, iEnd = this.levels; i < iEnd; i++) {
             const row = new HeaderRow(vp, i + 1); // Avoid indexing from 0
             row.renderMultipleLevel(i);
             this.rows.push(row);
         }
+
+        // Render a row with empty cells for filtering.
+        if (this.columns.some((column): boolean =>
+            !!column.options?.filtering?.enabled
```

Comment:
Do we want to introduce the concept of filtering type? I'm not sure if this should be set on the column tho.. 
Maybe `rendering.filtering.type = 'inline'`